canonical-web-and-design/tools.demos.haus
# Add form for generating social banner canvas

## Done

Built form to generate various social media images

## QA

- Checkout this feature branch
- Run the site using `dotrun`
- View the site locally in your browser at: [http://0.0.0.0:8057](http://0.0.0.0:8057)
- Check that this URL renders the social images: http://localhost:8057/?title=Ubuntu+for+the+internet+of+things&subtitle=From+smart+homes+to+smart+drones%2C+robots%2C+and+industrial+systems%2C+Ubuntu+is+the+new+standard+for+embedded+Linux.&illustration_url=https%3A%2F%2Fassets.ubuntu.com%2Fv1%2F049b4cc6-ubuntu%2Bcore%2B18%2Bcircular%2Bwhite.svg&background=dark
- Try changing the text and background and submitting the form and check that the images are updated

**Note**: The text styles, illustration size and gradients aren't right but that is not what this PR addresses

## Issue / Card
Fixes #5
